tests/virt_autotest/sriov_network_card_pci_passthrough.pm

```diff
@@ -160,7 +167,7 @@ sub prepare_host {
 
     #check VT-d is supported in Intel x86_64 machines
     if (script_run("grep Intel /proc/cpuinfo") == 0) {
-        assert_script_run "dmesg | grep -E \"DMAR:.*IOMMU enabled\"";
+        assert_script_run "dmesg | grep -E \"DMAR:.*IOMMU enabled|iommu.*Passthrough\"";
```

Comment:
`iommu.*Passthrough` does not flag that `intel_iommu=on`, it only indicate `iommu=on`. In the meantime, `intel_iommu=on` is required for sriov tests, so this change is not correct. I described this issue in a poo ticket.  progress.opensue.org is inaccessible now, I am unable to link it at here now, but you can find it in your mailbox as I involved you in that ticket.

Reply:
progress.opensuse.org is available, here is the investigation I did, https://progress.opensuse.org/issues/151156?issue_count=289&issue_position=2&next_issue_id=151133&prev_issue_id=151162#note-1

Reply:
Ah, I got it.

Reply:
Added iommu boot parameter in autoyast and changed the code back